### PR TITLE
We don't support VolumesFrom or links

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@umohnani8 PTAL

**Comment by umohnani8** [comment]
LGTM

**Comment by mheon** [comment]
Git validation is complaining because you have trailing whitespace on your commit message

**Comment by rhatdan** [comment]
That is why  you never use vi to edit, emacs cleans this up automatically.  Repushed.

**Comment by mheon** [comment]
LGTM

File: docs/kpod-create.1.md [hunk 1]
```diff
@@ -307,9 +307,8 @@ unit, `b` is used. Set LIMIT to `-1` to enable unlimited swap.
 
    kpod generates a UUID for each container, and if a name is not assigned
 to the container with **--name** then the daemon will also generate a random
-string name. The name is useful when defining links (see **--link**) (or any
-other place you need to identify a container). This works for both background
-and foreground containers.
+string name. The name is usefull any place you need to identify a container.
```
Comment (TomSweeneyRedHat) [inline docs/kpod-create.1.md]: usefull to useful

**Comment by rhatdan** [comment]
@rh-atomic-bot r=mheon

**Pull request closed without merging**